Source/DataSources/CzmlDataSource.js

```diff
@@ -600,7 +600,10 @@ define([
         var hasInterval = defined(combinedInterval) && !combinedInterval.equals(Iso8601.MAXIMUM_INTERVAL);
 
         if (!isReference) {
-            referenceFrame = defaultValue(ReferenceFrame[packetData.referenceFrame], undefined);
+            if (defined(packetData.referenceFrame)) {
+                referenceFrame = ReferenceFrame[packetData.referenceFrame];
+            }
+            referenceFrame = defaultValue(referenceFrame, ReferenceFrame.FIXED);
```

Comment:
The test is against two different values.  The `if (defined())` checks whether the CZML has a value for `referenceFrame`, and then `defaultValue` checks whether the value in CZML is one of the valid options in the `ReferenceFrame` enum.  (the second case would be a good place for some validation logging or something, but we don't do any validation of CZML currently)